src/item/Item.php

```diff
@@ -558,6 +559,13 @@ public function onAttackEntity(Entity $victim, array &$returnedItems) : bool{
 		return false;
 	}
 
+	/**
+	 * Called when this item is being worn by an entity.
+	 */
+	public function onTickWorn(Living $entity) : bool{
+		return false;
```

Comment:
Please document the significance of the return value.

Reply:
This returns true in TurtleHelmet, which isn't changing its count or durability, so this doc isn't accurate.